### ISSUE TITLE
How to set different options for two calendars in one page?

### ISSUE THREAD

**Issue by tomasklima** [nekken/ng2-fullcalendar#14]
I have two calendars inserted to one page (in separate sibling components). Every calendar has its own options:

```
<angular2-fullcalendar [options]="calendar1Options" id="calendar1"></angular2-fullcalendar

<angular2-fullcalendar [options]="calendar2Options" id="calendar2"></angular2-fullcalendar>
```

But both calendars are set up by options of the first one.

**Comment by stevie1706** [comment]
In your component ts, you just need to different objects - as you have a above?

Post your component ts and your html.

**Comment by tomasklima** [comment]
Yes, I need because both of calendars have different settings. The first calendar shows only month and week views, the second one displays only day (actually selected in the first calendar). I have no sources in this time because I solved the problem using ability to set the options dynamically. but it needs a bit more coding - so setting the different options in declarative way would be useful.
The root of the problem is that [the CalendarComponent uses element selector to find the element in DOM](https://github.com/nekken/ng2-fullcalendar/blob/master/src/calendar/calendar.ts#L32). If it used the ID selector, the problem would not occur.

**Comment by shah-smit** [comment]
@tomasklima  were you able to load full-calendar? I am unable to, I am getting for 404 [error](http://stackoverflow.com/questions/42470498/xhr-error-404-not-found-loading-angular2-fullcalendar)!

**Comment by tomasklima** [comment]
@shah-smit I have no problem to load both calendars. I'm just dealing with impossibility to set calendar options in declarative way.

**Comment by nikopolv** [comment]
Any fixes or forks available for this?

**Comment by kurmeroli** [comment]
I got the same problem

**Comment by piyush-daffodil** [comment]
Hi , 

I've got similar problem , my requirement is to show 2 calendars at a time for current month and next month first and then have a "load more" button , on click of which it will and further 2 months calendar , settings will remain same for all the calendars but events will be different for each month.

is it possible . If yes , Please let me know how .
I am using it in angular 2. 

Thanks in advance.

**Comment by ricavir11** [comment]
Same problem.
Any fix planned ?

**Comment by landervs** [comment]
This was an issue, we have fixed it for our usage, to share it with everyone I created a pull request: https://github.com/nekken/ng2-fullcalendar/pull/70
